backend/djangoindia/db/models/event.py

```diff
@@ -26,32 +26,41 @@ class Event(BaseModel):
     venue = models.TextField(default="TBA",null=True, blank=True)
     city = models.CharField(max_length=255, default="TBA", null=True, blank=True)
     venue_map_link = models.TextField(null=True, blank=True)
-    event_start_date = models.DateTimeField(null=False, default=default_start_date)
-    event_end_date = models.DateTimeField()
+    start_date = models.DateTimeField(null=False, default=default_start_date)

```

Comment:
I find:  

- default start and end date increases chances of error. 
- Keep them Blank but required will force event creator to add a specific date 
- no chance of them accidentally using defaults, but with default chances of human error are present.

Reply:
?

Reply:
The idea behind that was to prevent creating events in past, but I think i should switch them with validators.